app/assets/scripts/views/locations-hub/locations-hub.js

```diff
@@ -58,7 +58,9 @@ export default function LocationsHub({
     setPage(() => getPage(query));
 
     setFilters({
-      order_by: query.order_by && query.order_by.split(','),
+      // In the front end we are using param 'area', but this is
+      // mapped to 'city' before getting sent to backend.
```

Comment:
Maybe we can consider the renaming on the backend as well, for better consistency? Or is there a specific reason it needs to be called city in the backend?